subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -12,133 +12,114 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+[[gradle_overview]]
 [[what_is_gradle]]
 = What is Gradle?
 
-[[gradle_overview]]
-== Overview
-
-Gradle is an open-source https://en.wikipedia.org/wiki/Build_automation[build automation] tool that is designed to be flexible enough to build almost any type of software. The following is a high-level overview of some of its most important features:
+Gradle is an open-source https://en.wikipedia.org/wiki/Build_automation[build automation] tool flexible enough to build almost any type of software.
+This flexibility is the result of Gradle making few assumptions about what you're trying to build or how to build it.
+Gradle bases its design on the following fundamentals:
 
 High performance::
-Gradle avoids unnecessary work by only running the tasks that need to run because their inputs or outputs have changed. You can also use a build cache to enable the reuse of task outputs from previous runs or even from a different machine (with a shared build cache).
-+
-There are many other optimizations that Gradle implements and the development team continually work to improve Gradle's performance.
+Gradle avoids unnecessary work by only running tasks that need to run because inputs or outputs changed.
+Gradle uses various caches to reuse outputs from previous runs.
+With a shared build cache, you can even reuse outputs across multiple machines.
```

Comment:
"have changed" reads better to me here, but I can't say if that's more correct.

```suggestion
Gradle avoids unnecessary work by only running tasks that need to do work because inputs or outputs have changed.
Gradle uses various caches to reuse outputs from previous builds.
With a shared build cache, you can even reuse outputs from other machines.
```